energy-integration/governance-v2/src/configurable.rs

```diff
@@ -152,4 +171,8 @@ pub trait ConfigurablePropertiesModule: energy_query::EnergyQueryModule {
     #[view(getLockTimeAfterVotingEndsInBlocks)]
     #[storage_mapper("lockTimeAfterVotingEndsInBlocks")]
     fn lock_time_after_voting_ends_in_blocks(&self) -> SingleValueMapper<u64>;
+
+    #[view(getGovernanceTokenId)]
+    #[storage_mapper("governanceTokenId")]
+    fn governance_token_id(&self) -> SingleValueMapper<TokenIdentifier>;
```

Comment:
Should be `fee_token_id` instead. `governance_token_id` makes me think it's used for voting.